src/Microsoft.AspNet.SignalR.Core/Messaging/MessageBus.cs

```diff
@@ -152,7 +152,7 @@ public virtual Task Publish(Message message)
             if (Topics.TryGetValue(message.Key, out topic))
             {
                 topic.Store.Add(message);
-                ScheduleTopic(message.Key, topic);
```

Comment:
ScheduleTopic had unused parameter. Needed to fix this because code would not compile due to an error. Not sure why it did not show up before.